### PR TITLE
Pull In DotENV for Environment Configuration

### PR DESCRIPTION
**Pull request by WormJim**

### CODE REVIEW

**Comment by WormJim** [comment]
@junjo255 Please Review

**Comment by junjo255** [comment]
Looks good ~

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "bootstrap": "^4.2.1",
     "compression": "^1.7.3",
     "cors": "^2.8.5",
+    "dotenv": "^6.2.0",
     "express": "^4.16.4",
```
Comment (junjo255) [inline package.json:23]: Cool!

**Review by junjo255 — APPROVED** [review]
Everything looks good. Please, review my comment.

File: package.json (hunk 1 above)
Comment (WormJim) [reply package.json:23]: Great, I'll merge!

**Pull request merged**